ansible_collections/arista/avd/roles/eos_designs/templates/network_services/vlan-interfaces.j2

```diff
@@ -142,7 +142,11 @@ vlan_interfaces:
     shutdown: false
     description: "MLAG_PEER_L3_iBGP: vrf {{ vrf.name }}"
     vrf: {{ vrf.name }}
+{%                     if underlay_rfc5549 is arista.avd.defined(true) and switch.underlay_ipv6 is arista.avd.defined(true) %}
+    ipv6_enable: true
+{%                     else %}
     ip_address: {{ switch.mlag_l3_ip | arista.avd.default(switch.mlag_ip) }}/31
+{%                     endif %}
```

Comment:
Question - are v6 and ipv4 necessarily mutually exclusive? or could you have both?

Reply:
With RFC5549 you should normally not use ipv4 addresses on the interfaces. We did not have this in the original rfc5549 implementation, since EOS support in VRFs was not clear to us. Recently we received "blessing" from the devs, so now we can improve the rfc5549 implementation.
There is no real relationship with underlay_ipv6, so I think I will implement a `underlay_rfc5549_mlag` knob to enable this behavior, to avoid breaking changes.

Reply:
overlay_mlag_rfc5549 added instead of the underlay_ipv6 check